package.json

```diff
@@ -39,9 +39,9 @@
     "immutable": "3.7.6",
     "microtime-fast": "1.0.2",
     "morgan": "1.7.0",
+    "promise-sqlite": "1.1.0",
     "request": "2.65.0",
     "source-map-support": "ncalexan/node-source-map-support#fileUrls-plus",
-    "sqlite3": "mossop/node-sqlite3#v3.1.4.1",
```

Comment:
@Mossop suggests this may cause problems with `electron-rebuild`, but I don't know the details.